README.md

```diff
@@ -282,7 +282,8 @@ $ gem install google-cloud-language
 ```ruby
 require "google/cloud/language"
 
-language = Google::Cloud::Language.new
+language = Google::Cloud::Language.new( project: "my-todo-project",
+                                        keyfile: "/path/to/keyfile.json" )
 
```

Comment:
The existing manually-written examples showing initialization with `project` and `keyfile` all place the args on new lines: [BigQuery](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-bigquery/README.md), [Datastore](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-datastore/README.md), [Pub/Sub](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-pubsub/README.md) and [Storage](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-storage/README.md). (Debugger is an exception, but it has a different origin, and it should be changed.)